Mathlib/LinearAlgebra/SesquilinearForm.lean

```diff
@@ -183,45 +183,77 @@ end Reflexive
 
 /-! ### Symmetric bilinear forms -/
 
+section ConjugateSymmetric
 
-section Symmetric
+variable [AddCommMonoid M₁] [AddCommMonoid M] [CommSemiring R] [Module R M₁] [Module R M]
+  {I : R →+* R} {J : M →+ M} {B : M₁ →ₛₗ[I] M₁ →ₗ[R] M}
 
-variable [CommSemiring R] [AddCommMonoid M] [Module R M] {I : R →+* R} {B : M →ₛₗ[I] M →ₗ[R] R}
+/-- The proposition that a sesquilinear form is conjugate symmetric -/
+def IsConjSymm (B : M₁ →ₛₗ[I] M₁ →ₗ[R] M) (J : M →+ M) : Prop :=
```

Comment:
I worry this is more generality than we really care about, and maybe we just want to allow `I = J = star`. Rather than changing anything here, it's probably best to discuss on Zulip.

Reply:
[Zulip Discussion](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Linear.2EIsSymm.20for.20sesquilinear.20maps)